tests/monotouch-test/CoreGraphics/CGImagePropertiesGPSTest.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using Foundation;
+#if MONOMAC
+using AppKit;
+#else
+using UIKit;
+#endif
+using CoreGraphics;
+using NUnit.Framework;
+using System.IO;
+
+namespace monotouchtest.CoreGraphics {
+	[TestFixture]
+	public class CGImagePropertiesGPSTest {
+		[Test]
+		public void LongitudeRefAndLattitudeRefTest ()
+		{
+			var keys = new NSString [] {
+				new NSString ("LatitudeRef"),
+				new NSString ("Latitude"),
+				new NSString ("LongitudeRef"),
+				new NSString ("Longitude")
+			};
+			var values = new object [] {
+				new NSString ("N"),
+				47.6422f,
+				new NSString ("W"),
+				-122.1367f
+			};
+			CGImagePropertiesGps gps = new (NSDictionary.FromObjectsAndKeys (values, keys));
+			Assert.AreEqual (gps.LatitudeRef, values [0]);
```

Comment:
For the assert statements, it might be a good idea to add an extra argument to provide a message in case an exception is thrown-  it's helpful to have the extra context in case the test fails.
see docs [here](https://learn.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.testtools.unittesting.assert.areequal?view=visualstudiosdk-2022#microsoft-visualstudio-testtools-unittesting-assert-areequal(system-object-system-object-system-string))